packages/react-components/src/components/styles/ImageGallery.style.ts

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IIconProps, mergeStyles, Theme } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const cancelIcon: IIconProps = { iconName: 'Cancel' };
+
+/**
+ * @private
+ */
+export const downloadIcon: IIconProps = {
+  iconName: 'Download'
+};
+
+/**
+ * @private
+ */
+export const overlayStyles = {
+  root: { background: 'rgba(0, 0, 0, 0.85)' }
+};
+
+/**
+ * @private
+ */
+export const focusTrapZoneStyle = mergeStyles({
```

Comment:
Instead of using `mergeStyles` everywhere can we strongly type these styles so they aren't misused elsewhere? Fluent provides style types for their components like: `IFocusTrapZoneStyles` or `IStackStyles`